x/icqoracle/keeper/query.go

```diff
@@ -120,6 +120,7 @@ func (k Keeper) TokenPriceToTokenPriceResponse(ctx sdk.Context, tokenPrices ...t
 			OsmosisPoolType:     tokenPrice.OsmosisPoolType,
 			SpotPrice:           tokenPrice.SpotPrice,
 			LastRequestTime:     tokenPrice.LastRequestTime,
+			LastResponseTime:    tokenPrice.LastResponseTime,
```

Comment:
ahh! 

I'm starting to think we should just change the response struct to be:
```
tokenPriceResult:
   tokenPrice: 
     ... struct in data
   unwrappedBaseDenom
   unwrappedQuoteDenom
```

Then we wouldn't have to keep making sure these were in sync

Reply:
good idea

Reply:
its just annoying that it's not at the same indentation level